docs/starters.yml

```diff
@@ -1149,3 +1149,35 @@
     - Netlify Identity
     - Netlify Functions
     - Static Marketing pages and Dynamic Client-side Authenticated App pages
+- url: https://gatsby-tutorial-starter.netlify.com/
+  repo: https://github.com/justinformentin/gatsby-v2-tutorial-starter
+  description: Simple, modern desgined blog with post lists, tags, and easily customizable code.
+  tags:
+    - Blog
+    - Prettier
+    - PWA
+    - SEO
+    - Styling:CSS-in-JS
+    - Linting
+    - Markdown
+  features:
+    - Blog post listing with image, summary, date, and tags.
+    - Post Tags
+    - Post List Filtering
+    - Typography.js
+    - Emotion styling
+    - Syntax Highlighting in Code Blocks
+    - Gatsby Image
+    - Fully Responsive
+    - Offline Support
+    - Web App Manifest
+    - SEO
+    - PWA
+    - Sitemap generation
+    - Schema.org JSON-LD
+    - CircleCI Integration
+    - Codeclimate Integration
+    - Google Analytics
+    - Twitter and OpenGraph Tags
+    - ESLint
+    - Prettier Code Styling
```

Comment:
Ironically that's the error :)

Reply:
Oh god, no way haha. Sorry about that. Thank you for fixing it.